jack.py

```diff
@@ -600,6 +680,14 @@ def transport_locate(self, frame):
         _check(_lib.jack_transport_locate(self._ptr, frame),
                "Error locating JACK transport")
 
+    def transport_query(self):
+        """return current transport position."""
+        # Test rolling transport status
+        if _lib.jack_transport_query(self._ptr, self.position) == 1:
+            return self.position
+        else:
+            return None
```

Comment:
Do you really want to return `None`?
Wouldn't raising an exception be more appropriate?
You could use the `_check()` function for that.